src/tcgwars/logic/impl/gen4/Unleashed.groovy

```diff
@@ -1896,8 +1896,8 @@ public enum Unleashed implements LogicCardInfo {
             energyCost D, D, C, C
             onAttack {
               damage 120
-              afterDamage{
-                my.deck.sublist(0,min(3, my.deck.size())).discard()
+              afterDamage {
+                my.deck.subList(0,Math.min(3, my.deck.size())).discard()
```

Comment:
```suggestion
                if (my.deck) my.deck.subList(0,Math.min(3, my.deck.size())).discard()
```

Not sure if this is needed (although it's used in a number of similar cards), but just in case.